site/streams.md

```diff
@@ -224,6 +224,46 @@ but some have some queue specific behaviour.
  * [Consumer acknowledgements](./confirms.html) (keep [QoS Prefetch Limitations](#global-qos) in mind)
  * Cancellation of consumers
 
+### <a id="single-active-consumer" class="anchor" href="#single-active-consumer">Single Active Consumer</a>
+
+Single active consumer for streams is a feature available in RabbitMQ 3.11 and more.
+It provides _exclusive consumption_ and _consumption continuity_ on a stream.
```

Comment:
"Single active consumer for streams is a feature available in RabbitMQ 3.11 and more."`

What does "and more" mean above??? later release to 3.11? If its in 3.11, it will be in later releases by default? Delete "and more" or else clarify further -thanks

Reply:
This is related to streams not part of this PR about Oauth 2.0